scripts/release.mjs

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env zx
+import { argv, cwd } from 'process'
+
+await $`set -e`
+
+const DIR = __dirname
+
+await $`zx ${DIR}/grabthar-validate-git`
+await $`zx ${DIR}/grabthar-validate-npm`
+
+let DIST_TAG = argv.find(element => element.includes('DIST_TAG='))?.replace('DIST_TAG=', '')
```

Comment:
Removed a previous comment cause `minimist` is applied to argv by default. I suspect we can handle args without iterating, but I might need to tinker a bit to be sure.

Reply:
Cool. Yeah, if we wanted to enforce a specific order for args, then we could simply use `argv[i]`, but I was thinking it'd be nice to leave it open to any order. I'm definitely open to suggestions, since I use this pattern several times (I could even grab the needed variables at one time if we want to combine them into a single operation).

Reply:
Yeah I think we can parse them into a map like structure with zx actually so potentially no need to worry about ordering

Reply:
@westeezy, whataya think of these apples? I can't defend this code as being any better than what I already had, other than it being DRYer:

```
let args = {};

for (const element of argv) {
  if (element.includes('=')) {
    const arg = element.split('=');
    args[arg[0]] = arg[1];
  }
};

let { DIST_TAG, BUMP, NPM_TOKEN } = args;

DIST_TAG ?? (DIST_TAG = 'latest');
BUMP ?? (BUMP = 'patch');
NPM_TOKEN ?? (NPM_TOKEN = '');
```

Reply:
I just implemented minimist `argv`. Nice and simple that way. 👍